boards/arm/nucleo_l552ze_q/doc/nucleol552ze_q.rst

```diff
@@ -276,14 +276,30 @@ You should see the following message on the console:
 Building a secure/non-secure with Arm |reg| TrustZone |reg|
 -----------------------------------------------------------
 
-The TF-M integration sample :ref:`tfm_ipc` can be run by a Nucleo L552ZE Q,
-using the ``nucleo_l552ze_q_ns`` target. When building a ``*_ns`` image with TF-M,
-a ``build/tfm/install/postbuild.sh`` bash script will be run as a post-build step
-to make some required flash layout changes. The ``build/tfm/regression.sh`` script
-will need to be run to perform device initialization, and then run ``west flash --hex-file build/tfm_merged.hex``
-to flash the board.
-
-Check the ``build/tfm/`` directory to ensure that the commands required by these scripts
+The TF-M integration sample :ref:`tfm_ipc` can be run on a ST Nucleo L552ZE Q.
+In TFM configuration, Zephyr is run on the non-secure side. A non-secure image
+could be generated using ``nucleo_l552ze_q_ns`` as build target.
+
+.. code-block:: bash
+
+   $ west build -b nucleo_l552ze_q_ns
```

Comment:
I assume the sample project path `samples/tfm_integration/tfm_ipc` is required in the build command?

Reply:
Sure, I'm adding path/to/source/directory (as this is board documentation, let's make it generic)

Reply:
Looks good to me.